django_freeradius/admin.py

```diff
@@ -66,6 +67,11 @@ class RadiusBatchAdmin(AbstractRadiusBatchAdmin):
     pass
 
 
+@admin.register(RadiusToken)
```

Comment:
I think this can be shown only when `DEBUG` is `True`, because normally administrators won't have to deal with this